### PR TITLE
add VLM support in llm bench

### PR DESCRIPTION
**Pull request by eaidova**
TO DO:

- [x] add test
- [x] check correctness of num_input_tokens after https://github.com/openvinotoolkit/openvino.genai/pull/1317
- [x] move unsupported pipelines to optimum

### CODE REVIEW

File: tools/llm_bench/llm_bench_utils/ov_utils.py [hunk 1]
```diff
@@ -515,6 +517,83 @@ def create_speech_2txt_model(model_path, device, **kwargs):
...
+            model_config = AutoConfig.from_pretrained(model_path, trust_remote_code=True)
+            remote_code = True
+        if kwargs.get("genai", True) and is_genai_available(log_msg=True):
+            if model_config.model_type.replace("_", "-") in GENAI_SUPPORTED_VLM:
+                log.info("Selected OpenVINO GenAI for benchmarking")
+                return create_genai_image_text_gen_model(model_path, device, ov_config, **kwargs)
+            else:
+                log.warning(
+                f"Model type `{model_config.model_type}` is not supported by OpenVINO GenAI. "
+                "Benchmark will be switched to Optimum Intel pipeline realization")
```
Comment (eaidova) [inline tools/llm_bench/llm_bench_utils/ov_utils.py:578]:
```suggestion
                log.warning(
                    f"Model type `{model_config.model_type}` is not supported by OpenVINO GenAI. "
                    "Benchmark will be switched to Optimum Intel pipeline realization"
                )

```

**Review by sbalandi — APPROVED** [review]

File: tools/llm_bench/task/image_text_to_text_generation.py [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+def load_image_genai(image_path):
+    pil_image = load_image(image_path)
+    image_data = np.array(pil_image.getdata()).reshape(1, pil_image.size[1], pil_image.size[0], 3).astype(np.uint8)
+    return ov.Tensor(image_data)
+
+
+def run_image_text_generation_genai(
+    inputs, num, model, processor, args, iter_data_list, md5_list, prompt_index, streamer, model_precision, proc_id, mem_consumption
+):
+    set_seed(args['seed'])
```
Comment (sbalandi) [inline tools/llm_bench/task/image_text_to_text_generation.py:192]: if i understand correctly, set_seed doesn't make sense here, for genai it should be configured via gen_config

Comment (eaidova) [reply tools/llm_bench/task/image_text_to_text_generation.py:192]:
```suggestion
```

File: .github/workflows/llm_bench-python.yml [hunk 3]
```diff
@@ -137,12 +137,16 @@ jobs:
           optimum-cli export openvino --trust-remote-code --model openai/whisper-tiny ./ov_models/whisper-tiny
           python ./tools/llm_bench/benchmark.py -m ./ov_models/whisper-tiny --media multilingual_librispeech/data/mls_polish/train/audio/3283_1447_000/3283_1447_000000.flac -d cpu -n 1 --optimum
           python ./tools/llm_bench/benchmark.py -m ./ov_models/whisper-tiny --media multilingual_librispeech/data/mls_polish/train/audio/3283_1447_000/3283_1447_000000.flac -d cpu -n 1
+      - name: Text llava-next-tiny on Linux
+        run: |
+          optimum-cli export openvino --model katuni4ka/tiny-random-llava-next ./ov_models/tiny-random-llava-next
```
Comment (eaidova) [inline .github/workflows/llm_bench-python.yml:142]:
```suggestion
          optimum-cli export openvino --model katuni4ka/tiny-random-llava-next ./ov_models/tiny-random-llava-next --task image-text-to-text
```

**Pull request merged**